sessions/js-basics/dark-mode/js/index.js

```diff
@@ -0,0 +1,13 @@
+console.clear();
+
+/*
+Add some interactivity to this website.
+ - if the body has the class "dark", the page is displayed in dark mode
+ - query all 3 buttons with the 'data-js' attribute
+ - add an event listener to each button:
+    - "switch to dark mode" button always adds the class "dark" to the body
+    - "switch to light mode" button always remvoes the class "dark" from the body
+    - "toggle mode" button toggles the class "dark" on the body
+*/
```

Comment:
This part should be written inside a readme file. You could take a look at the readme files of the objects and arrays challenges #65.

Reply:
I am going to commit a readme file in a couple of minutes.